packages/contracts/src/utils/JwtVerifier.sol

```diff
@@ -0,0 +1,158 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.9;
+
+import "../interfaces/IJwtGroth16Verifier.sol";
+import {UUPSUpgradeable} from "@openzeppelin/contracts/proxy/utils/UUPSUpgradeable.sol";
+import {OwnableUpgradeable} from "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+import { strings } from "solidity-stringutils/src/strings.sol";
+
+struct EmailProof {
+    string domainName; // Domain name of the sender's email
+    bytes32 publicKeyHash; // Hash of the DKIM public key used in email/proof
```

Comment:
Can you update the comment appropriately?

Reply:
@shreyas-londhe Thank you point out it. I'll update it.